README.md

```diff
@@ -0,0 +1,78 @@
+# ReactOS Project
+
+Current version: __0.4.6__
+![ReactOS Logo](https://reactos.org/wiki/images/0/02/ReactOS_logo.png)
+
+## Quick Links
+- [Website](https://reactos.org)
+- [Wiki](https://reactos.org/wiki)
+- [JIRA Bug tracker](https://jira.reactos.org/issues)
+- [ReactOS Git mirror](https://git.reactos.org)
+
+## What is ReactOS?
+
+ReactOS™ is an Open Source effort to develop a quality operating system that is
+compatible with applications and drivers written for the Microsoft® Windows™ NT
+family of operating systems (NT4, 2000, XP, 2003, Vista, Seven).
+
+The ReactOS project, although currently focused on Windows Server 2003
+compatibility, is always keeping an eye toward compatibility with
+Windows Vista and future Windows NT releases.
+
+The code of ReactOS is licensed under [GNU GPL 2.0+](https://spdx.org/licenses/GPL-2.0+.html).
+
+## Building ReactOS
+
+To build the system it is strongly advised to use the _ReactOS Build Environment
+(RosBE)._ Up-to-date versions for Windows and for Unix/GNU-Linux are available
+from our download page at: http://www.reactos.org/wiki/Build_Environment/
+
+Alternatively one can use Microsoft Visual C++ (MSVC) version 2010+, together
+with separate installations of CMake and the Ninja build utility.
+
+#### Binaries
+
+To build ReactOS run `ninja`, or alternatively run
+`make` if you are using the Make utility, from the top directory.
+
+__NOTE:__ In the other examples listed in the following, similar modification
+holds if you are using the Make utility instead of Ninja.
+If you are using RosBE, follow on-screen instructions.
+
+#### Bootable images
+
+To build a bootable CD image run `ninja bootcd` from the
+top directory. This will create a CD image with a filename, `ReactOS.iso`, in
+the top directory.
```

Comment:
"top directory" seems ambiguous. It's not the top-level source dir, it's the "top-level" build dir, but you should never be in any sub-directories of the build dir anyway.